api/src/main/java/org/openmrs/api/impl/EncounterServiceImpl.java

```diff
@@ -254,51 +258,12 @@ public Encounter getEncounter(Integer encounterId) throws APIException {
 		}
 		
 		List<Encounter> encs = new Vector<Encounter>();
-		for (Patient p : Context.getPatientService().getPatients(null, identifier, null, false)) {
+		for (Patient p : Context.getPatientService().getPatients(identifier, null, null, false)) {
```

Comment:
It actually failed after the deprecated methods removal. So this change was the fix.